src/app/screens/Onboard/TestConnection/index.jsx

```diff
@@ -32,8 +94,61 @@ export default function TestConnection() {
         console.log(e);
         setErrorMessage(e.message);
       });
+  }
+
+  useEffect(() => {
+    loadAccountInfo();
   }, []);
 
+  function renderFaucet() {
+    return (
+      <Modal
+        closeTimeoutMS={200}
+        isOpen={showFaucet}
+        onRequestClose={closeFaucet}
+        contentLabel="Allowance Options"
+        style={customStyles}
+        overlayClassName="bg-black bg-opacity-25 fixed inset-0"
+        className="absolute rounded-lg bg-white w-full max-w-lg"
+      >
+        <div className="p-5 flex justify-between">
+          <h2 className="text-2xl font-bold">Get some Satoshi</h2>
+          <button onClick={closeFaucet}>
+            <img
+              className="w-6 h-6"
+              src={CrossIcon}
+              alt=""
+              aria-hidden="true"
+            />
+          </button>
+        </div>
+        <div className="p-5 border-t border-b border-gray-200">
+          <p className="mb-2">
+            To get started we send {faucetAmount} Satoshi to your wallet.
+            <br />
+            Please provide your email. We will notify you of updates (don't
+            worry, we also hate spam)
+          </p>
+          <div className="w-60">
+            <Input
+              name="uri"
+              type="text"
+              onChange={handleEmailChange}
+              required
+              placeholder="you@email.com"
+            />
+          </div>
+        </div>
+        <div className="flex justify-end p-5">
+          {faucetLoading && <Loading />}
```

Comment:
You can refactor this to: `{faucetLoading ? <Loading /> : <Button onClick={claimSats} label="Get Sats" primary />}`